Dockerfile

```diff
@@ -1,8 +1,7 @@
 ARG SOLANA_IMAGE
 # Install BPF SDK
-FROM solanalabs/rust:1.63.0 AS builder
-RUN rustup toolchain install nightly
-RUN rustup component add clippy --toolchain nightly
+FROM solanalabs/rust:1.64.0 AS builder
+RUN rustup component add clippy
 WORKDIR /opt
 ARG SOLANA_REVISION
 RUN sh -c "$(curl -sSfL https://release.solana.com/"${SOLANA_REVISION}"/install)" && \
```

Comment:
What version of Solana BPF compiler will be installed here? Will it changed if Solana updated it?

Reply:
No, there is no changes except rust version. You can compare its layers:
1.64 — https://hub.docker.com/layers/solanalabs/rust/1.64.0/images/sha256-68b5cbce566deaffe9b89ec39d215f1b2062d2dc924ebcd1d5f3c5905774c322?context=explore
1.63 — https://hub.docker.com/layers/solanalabs/rust/1.63.0/images/sha256-0a335aeea2e5d48bb394116151ffe4d0db084b2de6fbf835f82a7e049e489c49?context=explore